examples/lighting-app/lighting-common/gen/endpoint_config.h

```diff
@@ -1018,13 +1018,17 @@
             /* Endpoint: 0, Cluster: Operational Credentials (server) */                                                           \
             { 0x003E, 0x00, ZAP_COMMAND_MASK(INCOMING_SERVER) }, /* SetFabric */                                                   \
             { 0x003E, 0x01, ZAP_COMMAND_MASK(INCOMING_CLIENT) }, /* SetFabricResponse */                                           \
+            { 0x003E, 0x02, ZAP_COMMAND_MASK(INCOMING_SERVER) }, /* CertChainRequest */                                            \
+            { 0x003E, 0x03, ZAP_COMMAND_MASK(INCOMING_CLIENT) }, /* CertChainResponse */                                           \
             { 0x003E, 0x04, ZAP_COMMAND_MASK(INCOMING_SERVER) }, /* OpCSRRequest */                                                \
             { 0x003E, 0x05, ZAP_COMMAND_MASK(INCOMING_CLIENT) }, /* OpCSRResponse */                                               \
             { 0x003E, 0x06, ZAP_COMMAND_MASK(INCOMING_SERVER) }, /* AddOpCert */                                                   \
             { 0x003E, 0x08, ZAP_COMMAND_MASK(INCOMING_CLIENT) }, /* OpCertResponse */                                              \
             { 0x003E, 0x09, ZAP_COMMAND_MASK(INCOMING_SERVER) }, /* UpdateFabricLabel */                                           \
             { 0x003E, 0x0A, ZAP_COMMAND_MASK(INCOMING_SERVER) }, /* RemoveFabric */                                                \
             { 0x003E, 0x0B, ZAP_COMMAND_MASK(INCOMING_SERVER) }, /* RemoveAllFabrics */                                            \
+            { 0x003E, 0x0C, ZAP_COMMAND_MASK(INCOMING_SERVER) }, /* AttestationRequest */                                          \
+            { 0x003E, 0x0D, ZAP_COMMAND_MASK(INCOMING_CLIENT) }, /* AttestationResponse */                                         \
             { 0x003E, 0xA1, ZAP_COMMAND_MASK(INCOMING_SERVER) }, /* AddTrustedRootCertificate */                                   \
             { 0x003E, 0xA2, ZAP_COMMAND_MASK(INCOMING_SERVER) }, /* RemoveTrustedRootCertificate */                                \
                                                                                                                                    \
```

Comment:
The new commands are being added with incorrect IDs.

Please make the non-standard commands have different IDs and use spec-compliant IDs for the new code:

```
| Id | Name                         | Direction        | Response                  | Access | Conformance
| 0  | AttestationRequest           | Client => Server | AttestationResponse       | A      | M
| 1  | AttestationResponse          | Server => Client | Y                         | A      | M
| 2  | CertificateChainRequest      | Client => Server | CertificateChainResponse  | A      | M
| 3  | CertificateChainResponse     | Server => Client | Y                         | A      | M
| 4  | CSRRequest                   | Client => Server | CSRResponse               | A      | M
| 5  | CSRResponse                  | Server => Client | Y                         | A      | M
| 6  | AddNOC                       | Client => Server | NOCResponse               | A      | M
| 7  | UpdateNOC                    | Client => Server | NOCResponse               | A      | M
| 8  | NOCResponse                  | Server => Client | Y                         | A      | M
| 9  | UpdateFabricLabel            | Client => Server | NOCResponse               | A      | M
| 10 | RemoveFabric                 | Client => Server | NOCResponse               | A      | M
| 11 | AddTrustedRootCertificate    | Client => Server | Y                         | A      | M
| 12 | RemoveTrustedRootCertificate | Client => Server | Y                         | A      | M
```

Reply:
CD construction was removed - Updated.